format/postgres/pg_heap.go

```diff
@@ -0,0 +1,69 @@
+package postgres
+
+import (
+	"embed"
+
+	"github.com/wader/fq/format/postgres/common/pg_heap/pgproee"
+	"github.com/wader/fq/format/postgres/common/pg_heap/postgres"
+
+	"github.com/wader/fq/format"
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/interp"
+)
+
+// TO DO
+// oom kill on 1 GB file
+
+//go:embed pg_heap.md
+var pgHeapFS embed.FS
+
+func init() {
+	interp.RegisterFormat(decode.Format{
+		Name:        format.PG_HEAP,
+		Description: "PostgreSQL heap file",
+		DecodeFn:    decodePgheap,
+		DecodeInArg: format.PostgresHeapIn{
+			Flavour: PG_FLAVOUR_POSTGRES14,
+			Page:    0,
+			Segment: 0,
+		},
+		RootArray: true,
+		RootName:  "pages",
+	})
```

Comment:
See comment about `pg_btree.go` what to change, also same about `PostgresHeapIn` i guess, maybe rename to match format name

Reply:
It's done